modules/consignment-api/outputs.tf

```diff
@@ -14,3 +14,7 @@ output "database_password" {
   value     = aws_rds_cluster.consignment_api_database.master_password
   sensitive = true
 }
+
+output "api_url" {
+  value = "https://${aws_route53_record.api_dns.name}.${var.dns_zone_name_trimmed}"
```

Comment:
Out of curiosity, how does the frontend get this URL, and should it also use this output?

Reply:
The frontend has one application conf file per environment so the urls are all hard coded which is why we've never needed this before.